schema/defs-image.json

```diff
@@ -0,0 +1,99 @@
+{
+  "description": "Definitions particular to OpenContainer Image Specification",
+  "definitions": {
+    "mediaType": {
+      "id": "https://opencontainers.org/schema/image/mediaType",
+      "type": "string",
+      "pattern": "^[a-z]+/[0-9a-zA-Z.+]+$"
+    },
+    "digest": {
+      "description": "the cryptographic checksum digest of the object, in the pattern '<hash>:<hexadecimal digest>'",
+      "type": "string",
+      "pattern": "^[a-z0-9]+:[a-fA-F0-9]+$"
+    },
+    "blobObject": {
```

Comment:
This is what we have called it from the start. The godoc is [here](https://godoc.org/github.com/docker/distribution#Descriptor). I am not sure how it never made into this specification.

Other types with further fields, such as the manifest descriptor, are extensions to the descriptor type.